doc/ref_arch/kubernetes/chapters/appendix-a.md

```diff
@@ -1,21 +1,21 @@
 [<< Back](../../kubernetes)
-# Appendix A - Guidance for VNF-to-CNF Transition
+# Appendix A - Guidelines
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [A.1 TBC](#A.1)
-* [A.2 TBC](#A.2)
-* [A.3 TBC](#A.3)
-* [A.4 TBC](#A.3)
+* [A.1 Guidance for NFVI Vendors](#A.1)
+* [A.2 CNF Transition Plan](#A.2)
+* [A.3 Guidance for VNF-to-CNF Transition](#A.3)
+* [A.4 TBC](#A.4)
 
 <a name="A.1"></a>
-## A.1 TBC
+## A.1 Guidance for NFVI Vendors
```

Comment:
See above comment for line 6.

Reply:
Yes, NFVI is removed and left to the text underneath to explain.